contracts/interfaces/IPoolManager.sol

```diff
@@ -112,19 +113,11 @@ interface IPoolManager is IFees, IERC1155 {
     /// @notice Returns the reserves for a given ERC20 currency
     function reservesOf(Currency currency) external view returns (uint256);
 
-    /// @notice Contains data about pool lockers.
-    struct LockData {
-        /// @notice The current number of active lockers
-        uint128 length;
-        /// @notice The total number of nonzero deltas over all active + completed lockers
-        uint128 nonzeroDeltaCount;
-    }
-
     /// @notice Returns the locker in the ith position of the locker queue.
     function getLock(uint256 i) external view returns (address locker);
 
-    /// @notice Returns lock data
-    function lockData() external view returns (uint128 length, uint128 nonzeroDeltaCount);
+    /// @notice Returns the sentinel node for the LockData structure.
+    function getLockSentinel() external view returns (LockSentinel sentinel);
```

Comment:
whats the problem with continuing to call this `LockData`? I think I feel extremely opposed to the term "sentinel" as it's not descriptive and means [something else](https://en.wikipedia.org/wiki/Sentinel_value) in the context of computer programming :P